Atomic.md

```diff
@@ -0,0 +1,194 @@
+<pre>
+Title:       <b>Atomic/Batch Transactions</b>
```

Comment:
What do you think about just calling this `Batch`? IMHO, `Atomic` implies that all transactions will always work together (or not) but Batch leaves room for that not always holding.

Reply:
I actually perceived the connotation as reversed. The transactions are always executing atomically, but they're not always all executing.

Reply:
Oh interesting -- I was thinking more along the lines of `ACID` transactions (like, in a database) where it's something like "all of the commands that make up a transaction are treated as a single unit and either `all succeed` or `all fail` together."

Reply:
Honestly, the community has mostly been calling it `Batch`, so I'm not opposed to changing it. I've just gotten a bit attached to `Atomic` 🙂 . We'd probably also want to rename `AtomicityType` to `BatchType` then.

Reply:
I tried looking up what other chains call it and:
> On Algorand, atomic transfers are implemented as irreducible batch operations

🤦 
https://developer.algorand.org/docs/get-details/atomic_transfers/

Reply:
Renamed 🙂